### DISCUSSION TITLE
Upload diff screenshots to Cypress Dashboard as well as actual screenshot

### DISCUSSION THREAD

**Discussion by viktorgogulenko** [FRSOURCE/cypress-plugin-visual-regression-diff#163] [General]
I really impressed how good this plugin and easy to use - thanks for that! But just one thing which we would like to have - by some reason Cypress is uploading only actual screenshots and not any diffs so this is hard to understand what is actually happened there. As a workaround I've implemented a script which is uploading diff files to S3 AWS bucket and after build downloading these files to our CI and attaching them to the build as artifacts. But this is not so good as we need always to dig into job and look for these files. I configured to save files using this structure:
`imagesDir: './snapshots/snapshots_local',`
Maybe I configured something in the wrong way - how it works from your side? Are you able to see diff files on Cypress Dashboard after test runs?
<img width="434" alt="Screenshot 2022-10-20 at 18 03 05" src="https://user-images.githubusercontent.com/12098350/197001318-2d6d99a9-8a3a-4d28-a668-f021cec9df1e.png">

**Comment by FRSgit** [comment]
Hey @viktorgogulenko,

Just to make it clear - your issue is that the screenshots are not visible in the Cypress Dashboard? Or are we talking something else as well?

**Reply by viktorgogulenko** [reply]
Diff screenshots are not going to the dashboard and most of "actual" screenshots which were uploaded - broken there.

**Comment by donwelsh** [comment]
Hey @FRSgit, Yes I also do not see the diff screenshots at all on Cypress Cloud. So far the actual screenshots are working fine for me. I am on v3.3.10 of the plugin and the latest version of Cypress v12.16.0.

**Comment by FRSgit** [comment]
Hey!

Sorry for long delay in response.
I've answered similar question here: https://github.com/FRSOURCE/cypress-plugin-visual-regression-diff/discussions/223#discussioncomment-10643379

Unfortunately, I think it's not possible to provide better integration with Cypress Cloud
